crates/trustchain-http/tests/attestation.rs

```diff
@@ -1,29 +1,51 @@
 /// Integration test for attestation challenge-response process.
+use tokio::runtime::Runtime;
 use trustchain_core::verifier::Verifier;
 use trustchain_http::attestation_encryption_utils::{josekit_to_ssi_jwk, ssi_to_josekit_jwk};
 use trustchain_http::attestation_utils::{
     attestation_request_path, CRState, ElementwiseSerializeDeserialize, IdentityCRChallenge,
     IdentityCRInitiation,
 };
 use trustchain_http::attestor::present_identity_challenge;
+use trustchain_http::config::HTTPConfig;
 use trustchain_http::requester::{
     identity_response, initiate_content_challenge, initiate_identity_challenge,
 };
 
-use trustchain_http::utils::init_http;
 use trustchain_ion::{trustchain_resolver, verifier::TrustchainVerifier};
 
 // The root event time of DID documents used in integration test below.
 const ROOT_EVENT_TIME_1: u64 = 1666265405;
 
 use mockall::automock;
-use trustchain_core::utils::extract_keys;
+use trustchain_core::utils::{extract_keys, init};
 
 #[automock]
 pub trait AttestationUtils {
     fn attestation_request_path(&self) -> String;
 }
 
+fn init_http() {
+    init();
+    let http_config = HTTPConfig {
+        host: "127.0.0.1".parse().unwrap(),
+        port: 8081,
```

Comment:
Check error message is clear when 8081 is in use and add suggestion for fixing if not.

Reply:
Implemented in [63b6933](https://github.com/alan-turing-institute/trustchain/pull/220/commits/63b6933cefc2c4bd438aa335072c177bfe2e8388).